docs/articles/infrastructure/ci-cd/build-slaves.rst

```diff
@@ -88,7 +88,7 @@ Configuring Jenkins
     * 7c. Set "Username" to "jenkins" (assuming that you use this UNIX username for your jenkins user)
     * 7d. Set "Private Key" to "Enter Directly", and supply the *private* key you generated previously
     * 7e. Set "Passphrase" as needed, or leave empty if no passphrase is set
-    * 7f. Set "Description" to a suitable description
+    * 7f. Add a suitable description in the "Description"
```

Comment:
I think the previous format was better because it followed the same convention used in the rest of the list.